packages/utils/src/lib/report.unit.test.ts

```diff
@@ -182,3 +190,88 @@ describe('loadReport', () => {
     ).rejects.toThrow('validation');
   });
 });
+
+describe('sortCategoryAudits', () => {
+  it('should sort audits by weight and score', () => {
+    const mockAudits = [
+      { weight: 0, score: 0.1 },
+      { weight: 5, score: 1 },
+      { weight: 0, score: 0.7 },
+      { weight: 10, score: 1 },
+    ] as WeighedAuditReport[];
+    const sortedAudits = [...mockAudits].sort(sortCategoryAudits);
+    expect(sortedAudits[0]).toEqual(mockAudits[3]);
```

Comment:
It would increase the readability of the tests if you asserted the whole `sortedAudits` array and included the expected values directly.

Reply:
I haven't quite understood the proposition 🙂 
Would you like me to compare the whole array?
```suggestion
    expect(sortedAudits).toEqual([
      { weight: 10, score: 1 },
      { weight: 5, score: 1 }, // ...
    ]);
```
Or probably you'd like the values to be compared?
```suggestion
    expect(sortedAudits[0].weight).toEqual(10);
    expect(sortedAudits[0].score).toEqual(1); // comparing the values directly
```

Reply:
The whole array, yes. Then the person reading the test can tell straight away if the sorting result is valid (sees the values below each other).

Reply:
Done